packages/host/src/components/AppBar/SectionIcon.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  ListIcon,
+  LocaleKey,
+  matchPath,
+  ReportsIcon,
+  RouteBuilder,
+  StockIcon,
+  SuppliersIcon,
+  Tooltip,
+  TruckIcon,
+  useLocation,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+
+type Section = {
+  icon?: JSX.Element;
+  titleKey: LocaleKey;
+};
+
+const getIcon = (section?: AppRoute) => {
+  switch (section) {
+    case AppRoute.Catalogue:
+      return <ListIcon color="primary" fontSize="small" />;
+    case AppRoute.Distribution:
+      return <TruckIcon color="primary" fontSize="small" />;
+    case AppRoute.Inventory:
+      return <StockIcon color="primary" fontSize="small" />;
+    case AppRoute.Replenishment:
+      return <SuppliersIcon color="primary" fontSize="small" />;
+    case AppRoute.Reports:
+      return <ReportsIcon color="primary" fontSize="small" />;
+    default:
+      return undefined;
+  }
+};
+
+const getSection = (): Section | undefined => {
+  const routes = [
+    AppRoute.Catalogue,
+    AppRoute.Distribution,
+    AppRoute.Inventory,
+    AppRoute.Replenishment,
+    AppRoute.Reports,
+  ];
+  const location = useLocation();
+
+  for (let i = 0; i < routes.length; i++) {
+    const route = routes[i];
+    const match = matchPath(
+      RouteBuilder.create(route ?? '')
+        .addWildCard()
+        .build(),
+      location.pathname
+    );
+    if (!!match)
+      return {
+        icon: getIcon(route),
+        titleKey: route as LocaleKey,
```

Comment:
iffy, and possibly not needed - having a tooltip on the icon. I think it's helpful to explain them though, so put the tooltip on.